auth/src/main/java/org/aerogear/mobile/auth/credentials/JwksManager.java

```diff
@@ -0,0 +1,155 @@
+package org.aerogear.mobile.auth.credentials;
+
+
+import android.content.Context;
+import android.content.SharedPreferences;
+
+import org.aerogear.mobile.auth.configuration.AuthServiceConfiguration;
+import org.aerogear.mobile.auth.configuration.KeycloakConfiguration;
+import org.aerogear.mobile.core.executor.AppExecutors;
+import org.aerogear.mobile.core.http.HttpRequest;
+import org.aerogear.mobile.core.http.HttpResponse;
+import org.aerogear.mobile.core.http.HttpServiceModule;
+import org.aerogear.mobile.core.logging.Logger;
+import org.jose4j.jwk.JsonWebKeySet;
+import org.jose4j.lang.JoseException;
+
+import java.util.Date;
+
+/**
+ * A class that is responsible for manage the Json Web Key Set(JWKS).
+ */
+class JwksManager {
```

Comment:
public?  If it is supposed to be package protected maybe @PublicForTesting?

Reply:
yes, it should be public